src/less/RevisionPopupWidget.less

```diff
@@ -0,0 +1,29 @@
+.ext-wwt-revisionPopupWidget-attribution {
+	margin-top: 14px;
+}
+
+/* This matches the native MediaWiki CSS for diff sizes, which may not be available on article pages. */
+
+.mw-diff-bytes {
+	font-style: italic;
+
+	&::before {
+		content: '(';
```

Comment:
Don't put parentheses hard coded like this; it's messing up RTL, and won't work for interface languages that have different definition for parentheses, like French.

MW has a `parentheses` message, use that?

Reply:
(See my comment above.) We can probably use the mediawiki.special.changeslist module.

Reply:
No, not that one, oops. mediawiki.interface.helpers.styles :)